seekarc/build.gradle

```diff
@@ -26,8 +28,14 @@ android {
             assets.srcDirs = ['assets']
         }
     }
+
+    compileOptions {
+        encoding = 'UTF-8'
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
```

Comment:
Is this necessary? I don't think this module uses any lambdas and doesn't have any Unicode characters in Java code.